src/ecalc/cli/tests/snapshots/test_app/test_json_true/test_v3.json

```diff
@@ -787,6 +787,34 @@
         2000000.0,
         3000000.0
       ],
+      "requested_inlet_pressure": [
```

Comment:
Usually we specify in and outlet pressure, but with e.g. ASV it might be different from input. So, why are there so many empty/null values here? It's probably something I dont understand/see ... :)

Reply:
ok